server/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM us.gcr.io/grafeas/grafeas-server
+FROM us.gcr.io/grafeas/grafeas-server:0.1.0
```

Comment:
Is there a travis test that makes sure docker build works here?

Reply:
Added a CircleCI test, for consistency with Grafeas setup, that does `docker-compose up` and curls an endpoint.